src/tools/miri/tests/fail/dangling_pointers/dangling_pointer_deref_underscore.rs

```diff
@@ -0,0 +1,11 @@
+// Make sure we find these even with many checks disabled.
+//@compile-flags: -Zmiri-disable-alignment-check -Zmiri-disable-stacked-borrows -Zmiri-disable-validation
```

Comment:
```suggestion
// compile-flags: -Zmiri-disable-alignment-check -Zmiri-disable-stacked-borrows -Zmiri-disable-validation
```
or does that `@` have a special meaning?

Reply:
yea, in miri we actually validate all such flags. If you use `//@` an actual parser with errors will check your flags. If you don't use `@`, the flag is entirely ignored, it's just a comment then.